dev-haskell/record-dot-preprocessor/record-dot-preprocessor-0.2.7-r1.ebuild

```diff
@@ -15,6 +15,9 @@ SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
+IUSE="executable"
+
+PATCHES=( "${FILESDIR}/${P}-add-executable-flag.patch" )
```

Comment:
There's no need to change this, but I thought I'd mention it: there seems to be a preference to do things as follows:

```sh
PATCHES=( "${FILESDIR}/${PN}-0.2.7-add-executable-flag.patch" )
```

In other words, using `PN` instead of `P` and explicitly stating the version.

Why is this the preference?

I couldn't say for sure, but I imagine it's because it makes it less error prone to accidentally apply the wrong patch.

Again, no reason to change, and I see both used in ::gentoo.

Reply:
It just saves a bit of extra typing. If a patch fails, it will get caught during testing.